### PR TITLE
The test:all script runs on 0.13, 0.14 and 15

### PR DESCRIPTION
**Pull request by bettiolo**

### CODE REVIEW

**Comment by Aweary** [comment]
LGTM, thanks @bettiolo

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ npm test
 ```
 
 ```bash
-# run tests on both React 0.14 and React 0.13
```
Comment (ljharb) [inline CONTRIBUTING.md]: let's change this to "all supported versions of React" so we don't have to keep editing it as new versions come out :-)

**Comment by ljharb** [comment]
Awesome :-) it'd be nice to rebase but otherwise LGTM

**Comment by bettiolo** [comment]
@ljharb do I need to do anything?

**Comment by ljharb** [comment]
@bettiolo if you don't mind using a local git client and rebasing these down to 1 commit, that'd be great - if it's a problem, nbd tho

**Comment by bettiolo** [comment]
Done ;)

**Pull request merged**